Null resource object causes AssertionError

Given null to resource object, it causes AssersionError.

### version

0.2.1

### log
```
"AssertionError: assert(is_array($ro->body)) in /app/vendor/bear/qiq-module/src/QiqRenderer.php:33
```

### code

https://github.com/bearsunday/BEAR.QiqModule/blob/bbf5d97e2cbcc2fdd04e401204214b94f2c281df/src/QiqRenderer.php#L33

--- a/src/QiqRenderer.php
+++ b/src/QiqRenderer.php
@@ -30,8 +30,8 @@ final class QiqRenderer implements RenderInterface
     {
         $template = clone $this->template;
         $this->setView($template, $ro);
-        assert(is_array($ro->body));
-        $template->setData($ro->body);
+        assert(is_array($ro->body) || $ro->body === null);
+        $template->setData($ro->body ?? []);
 
         $ro->view = ($template)();
 

--- a/tests/Fake-src/Resource/FakeNullRo.php
+++ b/tests/Fake-src/Resource/FakeNullRo.php
@@ -1,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BEAR\QiqModule\Resource;
+
+use BEAR\Resource\ResourceObject;
+
+class FakeNullRo extends ResourceObject
+{
+    public function onGet(): ResourceObject
+    {
+        $this->body = null;
+
+        return $this;
+    }
+}

--- a/tests/Fake/templates/FakeNullRo.php
+++ b/tests/Fake/templates/FakeNullRo.php
@@ -1,0 +1,1 @@
+This resource has null body. Hello, World. That was Qiq! And this is PHP, World.

--- a/tests/QiqRendererTest.php
+++ b/tests/QiqRendererTest.php
@@ -4,6 +4,7 @@ declare(strict_types=1);
 
 namespace BEAR\QiqModule;
 
+use BEAR\QiqModule\Resource\FakeNullRo;
 use BEAR\QiqModule\Resource\FakeRo;
 use BEAR\QiqModule\Resource\SubDirectory\FakeSub;
 use BEAR\Resource\RenderInterface;
@@ -71,6 +72,17 @@ class QiqRendererTest extends TestCase
         $ro = $ro->onGet(['name' => 'World']);
         $view = (string) $ro;
         $this->assertSame('Hello, World. That was Qiq! And this is PHP, World.
+', $view);
+    }
+
+    public function testNullRender(): void
+    {
+        $ro = (new Injector($this->module))->getInstance(FakeNullRo::class);
+        assert($ro instanceof FakeNullRo);
+        $ro = $ro->onGet();
+        $view = (string) $ro;
+        $this->assertNotEmpty($ro->view);
+        $this->assertSame('This resource has null body. Hello, World. That was Qiq! And this is PHP, World.
 ', $view);
     }
 }

